src/migrations/migrations/1.js

```diff
@@ -1,8 +1,10 @@
+const db = require('db');
 const Migration = require('migrations/migration');
 
-const migration = new Migration(1, 'Example');
+const migration = new Migration(1, 'Set ttl index on tokens collection');
 
 migration.migrate = async () => {
+  await db.collection('tokens').createIndex({ expireAt: 1 }, { expireAfterSeconds: 0 });
```

Comment:
Remove from migrations.
Put this code inside tokenService itself (createIndex aka ensureIndex is idempotent)